src/schemas/secrets.schema.json

```diff
@@ -48,12 +48,12 @@
     "EMBED_SERVICE": {
       "type": "string",
       "description": "URL of an Embed Service that takes the appended URL and returns an embeddable HTML representation.",
-      "default": "https://adobeioruntime.net/api/v1/web/helix/helix-services/embed@v1"
+      "default": "https://helix-pages.anywhere.run/helix-services/embed@v1"
     },
     "DATA_EMBED_SERVICE": {
       "type": "string",
       "description": "URL of a DataEmbed Service that takes the appended URL and returns an iterable JSON representation.",
-      "default": "https://adobeioruntime.net/api/v1/web/helix/helix-services/data-embed@v1"
+      "default": "https://helix-pages.anywhere.run/helix-services/data-embed@v2"
```

Comment:
this doesn't really work for non-universal actions, since the version lock only works for runtime actions.
for universal, this is hardcoded:

https://github.com/adobe/helix-pipeline/blob/382707065e308b8cedd635b48b02a4555909b445/src/html/fetch-data.js#L29-L33

so I would revert the data-embed change

Reply:
I added a commit to revert the data-embed change